ncl/ncl_entries/general_applied_math.ipynb

```diff
@@ -0,0 +1,713 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    round_output = [round(value, 3) for value in input_values]
</pre><p>same comment here as with abs.</p><p> </p><p>and the other non-numpy functions below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Same here, I changed the array from a list comprehension to numpy, but kept it the same for single input from the standard library